extension/chrome/elements/compose-modules/formatters/encrypted-mail-msg-formatter.ts

```diff
@@ -212,8 +212,8 @@ export class EncryptedMsgMailFormatter extends BaseMailFormatter {
     });
   };
 
-  private formatEncryptedMimeDataAsPgpMimeMetaAttachments = (data: Uint8Array) => {
-    const attachments: Attachment[] = [];
+  private formatEncryptedMimeDataAsPgpMimeMetaAttachments = async (data: Uint8Array, pubkeys: PubkeyResult[]) => {
+    const attachments: Attachment[] = await this.view.attachmentsModule.attachment.collectEncryptAttachments(pubkeys);
```

Comment:
Seems like with such change we break standardized PGP/MIME message structure, which expects only 2 attachments (https://datatracker.ietf.org/doc/html/rfc3156#section-4).

Let's move attachments in password-protected messages into `encrypted.asc`, like in usual PGP/MIME messages.

It can be done on Monday, no rush to finish it on this week, as new browser extension release is expected on the next week.
Thanks for fixing all reported issues in time, well done!

Reply:
Yep!
You are right.
Let me implement as you suggested

Reply:
Fixed. please check